code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -172,6 +172,24 @@
 	color = BLOOD_COLOR_BUG // Bug colored, I guess.
 	pH = 7.25
 
+/datum/reagent/blood/tomato
+	data = list("donor"=null,"viruses"=null,"blood_DNA"=null, "bloodcolor" = BLOOD_COLOR_HUMAN, "blood_type"="SY","resistances"=null,"trace_chem"=null,"mind"=null,"ckey"=null,"gender"=null,"real_name"=null,"cloneable"=null,"factions"=null)
+	name = "Tomato Blood"
+	description = "This highly resembles blood, but it doesnt actually function like blood."
+	taste_description = "sap" //Like tree sap?
+	pH = 7.45
+
+/datum/reagent/blood/tomato/on_merge(list/mix_data) //Tomato blood doesnt mix.
+	return FALSE
+
+/datum/reagent/blood/tomato/on_mob_life(mob/living/carbon/C) //So it bloodsuckers cant use this as an unlimited blood supply
+	return
+
+/datum/reagent/blood/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
+	if(isvamp(L))
+		var/datum/antagonist/bloodsucker/bloodsuckerdatum = L.mind.has_antag_datum(ANTAG_DATUM_BLOODSUCKER)
+		bloodsuckerdatum.handle_eat_human_food(reac_volume * 2)
+	return
```

Comment:
This is going to override the previous `reaction_mob` function and fuck up virology.

Reply:
Fucked that up, whoops.